cinema/models.py

```diff
@@ -0,0 +1,17 @@
+from django.core.validators import MinValueValidator, MaxValueValidator
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=200)
+    description = models.TextField()
+    duration = models.IntegerField(
+        validators=[MinValueValidator(1), MaxValueValidator(300)]
+    )
+
+    class Meta:
+        verbose_name = "movies"
```

Comment:
why is it in plural? We use plural in `verbose_name_plural`
https://docs.djangoproject.com/en/4.2/ref/models/options/#verbose-name

Reply:
just typo, fixed now